static/docs/tutorial/define-ml-pipeline.md

```diff
@@ -17,7 +17,7 @@ to download `data.xml`and save it into the `data` subdirectory.
 
 ```dvc
 $ mkdir data
-$ wget -P data https://dvc.org/s3/so/100K/Posts.xml.zip
+$ wget -P data https://dvc.org/s3/data/tutorial-nlp/100K/Posts.xml.zip
```

Comment:
does it have 50K and 25K in it? do we use them?

Reply:
I actually didn't count them. I think it's 10 and 100 k based on the dirs inside the old bucket. See dvc-share/so and dvc-share/example/so. There's lots of files in there which aren't used in the docs, not sure whether we want to keep any or just back them up? For now I moved that I needed to dvc-public/data/tutorial-nlp 10K/ and 100K/

Reply:
I checked. It's 25K and 100K respectively. Updated bucket path, and the doc in 157e7b4.